files/grest/cron/jobs/pool-history-cache-update.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
+DB_NAME=cexplorer
```

Comment:
I guess if we are going to be using DB_NAME in multiple places we should move and source it from `env`?

Reply:
I see your point but not sure I want to pollute env with Koios specific variables.

Reply:
Ah, I see you are taking care of custom db names in  `setup-grest.sh` with `sed` instead.